addons/skin.estuary/xml/Variables.xml

```diff
@@ -147,7 +147,7 @@
 	<variable name="AddonsLabel2Var">
 		<value condition="ListItem.Property(addon.downloading)">$INFO[ListItem.Property(addon.status)]</value>
 		<value condition="!Container.SortMethod(1)">$INFO[ListItem.Label2]</value>
-		<value>$INFO[ListItem.AddonCreator,, - ]$INFO[ListItem.AddonVersion]</value>
+		<value>$INFO[ListItem.AddonOrigin,, - ]$INFO[ListItem.AddonVersion]</value>
```

Comment:
Does this only show in the all repos view?

Reply:
shows also in "per repo view" which makes no sense there.. not sure if the skin can control this

Reply:
You just need to figure out how to differentiate the all repos view from everything else. @jjd-uk or @ronie might know.

Reply:
you can use:
`<value condition="String.IsEqual(ListItem.Path,addons://all/)">$INFO[ListItem.AddonOrigin,, - ]$INFO[ListItem.AddonVersion]</value>`

and make sure to place it above the AddonCreator line.

Reply:
I'm still learning the GUI/skin stuff so this might be a very idiotic comment, but why are these labels not provided by Core via ListItem.Label2? With it done at skin level via `ListItem.AddonOrigin` we only have control over it being display by Estuary, so it's possible that none of the other skin authors will add it to their skins.